osu.Game/Screens/Play/HUD/ArgonSongProgress.cs

```diff
@@ -29,6 +30,9 @@ public partial class ArgonSongProgress : SongProgress
         [SettingSource(typeof(SongProgressStrings), nameof(SongProgressStrings.ShowTime), nameof(SongProgressStrings.ShowTimeDescription))]
         public Bindable<bool> ShowTime { get; } = new BindableBool(true);
 
+        [SettingSource(typeof(SkinnableComponentStrings), nameof(SkinnableComponentStrings.Colour), nameof(SkinnableComponentStrings.ColourDescription))]
+        public new BindableColour4 Colour { get; } = new BindableColour4(Colour4.White);
```

Comment:
Not a fan of the shadowing here (the usage of `new`) or the usage of `base` underneath. I'd rename all of these to `AccentColour` which gets rid of both.